Chrysalis Docs - Style Guide and Diataxis Review II (#114)
iotaledger/chrysalis-docs: docs/libraries/client.md

---

> description: The wallet library is a stateful library for developers, recommended if using IOTA for token transfers. This library has a stateful design and adds support for the handling and monitoring of multiple account/addresses.

Reviewer: ```suggestion
description: The wallet library is a stateful library for developers, recommended if you want to use IOTA for token transfers. This library has a stateful design and supports handling and monitoring multiple accounts and addresses.
```

> description: The wallet library is a stateful library for developers and it is recommended if you want to use IOTA for token transfers. This library has a stateful design and supports the handling and monitoring of multiple accounts and addresses.

Reviewer 2: ```suggestion
description: The wallet library is a stateful library for developers who want to use IOTA for token transfers. This library has a stateful design and supports the handling and monitoring of multiple accounts and addresses.
```

- Generate addresses.
- Interact with an IOTA node.

> However, if you are mainly processing value transfers, you should use our stateful [wallet library](wallet.md) instead.

Reviewer: ```suggestion
However, if you want to process value transfers, you should use our stateful [wallet library](wallet.md) instead.
```
